test/integration/gobierto_admin/gobierto_attachments/api/attachments_test.rb

```diff
@@ -201,7 +203,6 @@ def test_attachments_update_success
         assert_equal "Replace with new PDF file", db_pdf_attachment.name
         assert_nil db_pdf_attachment.description
         assert_equal "new-pdf-attachment.pdf", db_pdf_attachment.file_name
-        assert_equal "http://host.com/attachments/super-long-and-ugly-aws-id/new-pdf-attachment.pdf", db_pdf_attachment.url
```

Comment:
Why are you removing these checks? They were added for [this issue](https://github.com/PopulateTools/gobierto/pull/1394/files). Where they giving problems?

Reply:
Right, it was causing trouble and I forgot to restore the check.